default/python/AI/ProductionAI.py

```diff
@@ -1760,7 +1419,11 @@ def _location_rating(planet: fo.planet) -> float:
 
 
 def _try_enqueue(
-    building_type: BuildingType, candidates: Iterable[PlanetId], *, at_front: bool = False, ignore_dislike: bool = False
+    building_type: BuildingType,
+    candidates: Union[PlanetId, Iterable[PlanetId]],
```

Comment:
This change could be done in a separate PR. It's a nonbreaking change so you don't have to wait until the previous PR gets merged.  It was a really good opportunity to parallelize the review process. 

I encourage you to find a catch such cases in the future. 

- This makes PRs small, and smaller pr tend to have fewer comments. 
- You don't need to wait for feature A to be merged because of comments in Feature B.
- Refactoring of logic and refactoring of signatures require different levels of attention, when they are mixed, it draws a bit more reviewer attention to simple cases. 

This PR is already big, so it will take me a couple of passes to review it.  So moving such things to a separate PR will reduce that number.

Reply:
The problem was, that this extension occurred to me in the middle of all those changes, after writing  _try_enqueue(building_type, [pid]) several times.

My next PR should be parallel to this.

Reply:
It's not a problem.  It was before you submitted this PR. 

Here are simple git commands that could help you with it.

- git stash
- git checkout <new_branch>
- git add ...
- git commit
- git push 
- git checkout -
- git stash pop